tools/nni_cmd/tests/mock/restful_server.py

```diff
@@ -0,0 +1,186 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import responses
+
+def mock_check_status():
+    responses.add(
+        responses.GET,
+        "http://localhost:8080/api/v1/nni/check-status",
+        json={"status":"RUNNING","errors":[]},
+        status=200
+    )
+
+def mock_version():
+    responses.add(
+        responses.GET,
+        "http://localhost:8080/api/v1/nni/version",
+        json={'value':1.8},
+        status=200
+    )
+
+def mock_get_experiment_profile():
+    responses.add(
+        responses.GET,
+        "http://localhost:8080/api/v1/nni/experiment",
+        json={"id":"bkfhOdUl","revision":5,"execDuration":10,"logDir":"/home/shinyang/nni-experiments/bkfhOdUl",
+        "nextSequenceId":2,"params":{"authorName":"default","experimentName":"example_sklearn-classification",
+        "trialConcurrency":1,"maxExecDuration":3600,"maxTrialNum":1,
+        "searchSpace":"{\"C\": {\"_type\": \"uniform\", \"_value\": [0.1, 1]}, \
+        \"kernel\": {\"_type\": \"choice\", \"_value\": [\"linear\", \"rbf\", \"poly\", \"sigmoid\"]}, \
+        \"degree\": {\"_type\": \"choice\", \"_value\": [1, 2, 3, 4]}, \"gamma\": {\"_type\": \"uniform\", \
+        \"_value\": [0.01, 0.1]}}", \
+        "trainingServicePlatform":"local","tuner":{"builtinTunerName":"TPE","classArgs":{"optimize_mode":"maximize"}, \
+        "checkpointDir":"/home/shinyang/nni-experiments/bkfhOdUl/checkpoint"},"versionCheck":"true", \
+        "clusterMetaData":[{"key":"codeDir","value":"/home/shinyang/folder/examples/trials/sklearn/classification/."}, \
+        {"key":"command","value":"python3 main.py"}]},"startTime":1600326895536,"endTime":1600326910605},
+        status=200
+    )
+ 
+def mock_update_experiment_profile():
+    responses.add(
+        responses.PUT, 'http://localhost:8080/api/v1/nni/experiment',
+        json={"status":"RUNNING","errors":[]},
+        status=200,
+        content_type='application/json',
+    )
+
+def mock_import_data():
+    responses.add(
+        responses.POST, 'http://localhost:8080/api/v1/nni/experiment/import-data',
+        json={"result":"data"},
+        status=200,
```

Comment:
typical response status code for `POST` is `201` instead of `200`.

Reply:
fixed